lls/cache.c

```diff
@@ -498,11 +498,25 @@ lls_cache_init(struct lls_config *lls_conf, struct lls_cache *cache)
 	};
 
 	RTE_VERIFY(cache->key_len <= LLS_MAX_KEY_LEN);
+	cache->records = rte_calloc("lls_records",
+		lls_conf->lls_cache_records, sizeof(struct lls_record), 0);
+	if (cache->records == NULL) {
+		RTE_LOG(ERR, MALLOC, "Could not create %s cache records\n",
+			cache->name);
+		return -1;
+	}
+
 	cache->hash = rte_hash_create(&lls_cache_params);
 	if (cache->hash == NULL) {
 		RTE_LOG(ERR, HASH, "Could not create %s cache hash\n",
 			cache->name);
-		return -1;
+		goto records;
 	}
 	return 0;
+
+records:
+	rte_free(cache->records);
+	cache->records = NULL;
```

Comment:
`lls_cache_destroy()` needs to have these two lines as well for when the caches are destroyed.

Reply:
@mengxiang0811, did you address this comment?